3416.xml

```diff
@@ -0,0 +1,518 @@
+			</Item>
+			<Item ID="3">
+				<Name>Dimming level</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>0..100</RangeEnumeration>
+				<Units>/100</Units>
+				<Description>Dimming level (0 for OFF and 100% for ON) measured on the outdoor lamp controller.</Description>
+			</Item>
+			<Item ID="4">
+				<Name>Default dimming level</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>0..100</RangeEnumeration>
+				<Units>/100</Units>
+				<Description>The default dimming level that the outdoor lamp controller applies when the device is powered ON.</Description>
+			</Item>
+			<Item ID="5">
+				<Name>Lamp failure</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Set to True if the outdoor lamp controller detects that the lamp is not producing light while it is expected to (e.g. the lamp is broken).</Description>
+			</Item>
+			<Item ID="6">
+				<Name>Lamp failure reason</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Description of the reason why the lamp failed (e.g. low power on a LED engine, no consumption detected while relay closed).</Description>
+			</Item>
+			<Item ID="7">
+				<Name>Control gear failure</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Set to True in case the control gear has a failure. Outdoor lamp controllers may read the control gear failure from a DALI bus or by analyzing a 0-10 volts interface to the control gear.</Description>
+			</Item>
+			<Item ID="8">
+				<Name>Control gear failure reason</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>Description of the reason why the control gear failed. You may refer to the DiiA list of possible control gear failures.</Description>
```

Comment:
Reference (link) to the "DiiA list" would be great. Or at least enough information so that one can google the right list (unless it's obvious for everyone who would use this object)